src/fileimpl.cpp

```diff
@@ -599,6 +599,10 @@ class Grouping
     return zimReader->size();
   }
 
+  bool FileImpl::mayHaveAliasEntry() const {
+    return header.getMinorVersion() >= 2;
```

Comment:
Does ZIM format have version 5.2? Then this check is incorrect.
Besides it is not future-proof and will need to be updated when the major version is bumped.

Reply:
BTW, the same problem is present with the detection of whether the ZIM archive uses the new namespace scheme:

https://github.com/openzim/libzim/blob/abf7c11633038264385b521e75d7199fff18a94f/src/fileimpl.cpp#L359

Reply:
> Does ZIM format have version 5.2? 

No. Major version 6 is same as 5 but with extended cluster (blob bigger than for 4GB).
It has been use in parallel for a time then only 6.
Minor version was 0, then 1 for new namespace scheme (both with major 5 or 6) and now 2 (with 6 only, but it would be valid even with major 5)